sql/7_0_1-to-7_0_2_upgrade.sql

```diff
@@ -136,3 +136,24 @@ DROP TABLE IF EXISTS `pma_table_coords`;
 #IfTable pma_table_info
 DROP TABLE IF EXISTS `pma_table_info`;
 #EndIf
+
+#IfMissingColumn users_secure last_login_fail
+ALTER TABLE `users_secure` ADD `last_login_fail` datetime DEFAULT NULL;
+#EndIf
+
+#IfNotRow globals gl_name time_reset_password_max_failed_logins
+UPDATE `globals` SET `gl_value` = 20 WHERE `gl_name` = 'password_max_failed_logins' AND `gl_value` = 0;
+#EndIf
+
+#IfNotTable ip_tracking
+CREATE TABLE `ip_tracking` (
+`id` bigint NOT NULL auto_increment,
+`ip_string` varchar(255) DEFAULT '',
```

Comment:
This table will get large, does it make sense to store the ip string or store it as the 16 byte for ipv6 and 4byte for ipv4?  It will definitely be better for debugging/administration to have the string representation.

Reply:
note ip_string is not always a straightforward ip address. sometimes is 2 addresses in string form:
https://github.com/openemr/openemr/blob/9dc161d8d5b3fa71508b3e9f23b7c3e4aea49b97/library/sanitize.inc.php#L31